### PR TITLE
Use updated metrics.yaml only

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 GitPython
 boto3
-glean_parser
+glean_parser>=0.23.1
```
Comment (mdboom) [inline requirements.txt]: You probably want to make this `==0.23.1`.  There are some [very incompatible changes](https://github.com/mozilla/glean_parser/pull/58) coming along soon, and I'd hate to break `probe_scraper` before I have a chance to make a PR to fix it.

**Comment by fbertsch** [comment]
This will fix all those error emails.

**Comment by mdboom** [comment]
> This will fix all those error emails.

I don't think I get the error emails.  I should consider myself lucky?

File: requirements.txt (hunk 1 above)
Comment (fbertsch) [reply requirements.txt]: Cool, good idea.

Basically, we have those historical metrics files (from after 3-15 but before 4-12, when you removed `labeled` and added `labeled_*` metrics), which all failed with an error like below. Long-term we need to solve this with versioning:

Error in processing commit 5abfc5723d9730cb813137f8710d6dc857ee459a
Errors: [/mnt/analyses/probe-scraper/probe_cache/glean/5abfc5723d9730cb813137f8710d6dc857ee459a/components/service/glean/metrics.yaml:
    ```
    glean.error:
      invalid_label:
        bugs:
        - 1499761
        data_reviews:
        - https://bugzilla.mozilla.org/show_bug.cgi?id=1499761#c5
        description: Counts the number of times a metric was set with an invalid
          label. The labels are the `category.name` identifier of the metric.
        expires: never
        labeled: true
        notification_emails:
        - telemetry-client-dev@mozilla.com
        type: counter
    ```
    
    Additional properties are not allowed ('labeled' was unexpected)

Comment (mdboom) [reply requirements.txt]: Ah, thanks.

**Comment by fbertsch** [comment]
@mdboom added your email, and updated the requirements.

**Pull request merged**